WORKSPACE

```diff
@@ -341,14 +341,14 @@ nixpkgs_package(
 
 nixpkgs_package(
     name = "nixpkgs_lz4",
-    attribute_path = "lz4",
+    attribute_path = "lz4.out",
     build_file_content = """
 package(default_visibility = ["//visibility:public"])
 load("@rules_cc//cc:defs.bzl", "cc_library")
 
 cc_library(
   name = "nixpkgs_lz4",
-  srcs = glob(["lib/liblz4.dylib", "lib/liblz4.so*"]),
+  srcs = glob(["lib/liblz4.dylib", "lib/liblz4.so*"], allow_empty = True),
```

Comment:
It's not clear for me why `allow_empty` is needed here, could you clarify?

Reply:
It is because not only the result of the glob needs to be non-empty, but *all* patterns have to match something.

Depending on the OS, there will be `.so` files or `.dylib` but never both.

Naïvely, I tried to use `select()` on the `platform:os`, but since the dict given to select is evaluated eagerly, I ran into an empty glob error again.